typedspark/_schema/get_schema_definition.py

```diff
@@ -53,6 +54,7 @@ def _build_schema_definition_string(
             .replace("pyspark.sql.types.", "")
             .replace("typing.", "")
         )
+        typehint = _replace_literals_in_day_time_interval_type(typehint)
```

Comment:
Maybe this function name could be less datatype specific? Something like `_replace_literals_with_types`? It feels to me to be a strange place to have a reference to a very specific datatype. Or will day_time_interval be the only type that has to work with the literals in this way?